pull_request_template.md

```diff
@@ -15,52 +15,59 @@
 6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap
 
 ---
-##### Name (to be shown on DefiLlama): 
```

Comment:
please revert changes to this file and just put this information in a comment on the PR